libs/product/src/resolvers/product-page/product-page.resolver.ts

```diff
@@ -24,10 +25,7 @@ export class DaffProductPageResolver implements Resolve<Observable<boolean>> {
 		this.store.dispatch(new DaffProductLoad(route.paramMap.get('id')));
 
 		return this.dispatcher.pipe(
-			filter((action: DaffProductLoadSuccess | DaffProductLoadFailure) => 
-				action.type === DaffProductActionTypes.ProductLoadSuccessAction
-				|| action.type === DaffProductActionTypes.ProductLoadFailureAction
-			),
+			ofType(DaffProductActionTypes.ProductLoadSuccessAction || DaffProductActionTypes.ProductLoadFailureAction),
```

Comment:
Shouldn't it be `ofType(DaffProductActionTypes.ProductLoadSuccessAction, DaffProductActionTypes.ProductLoadFailureAction)`?

Reply:
yea, that does seem like the intended usage. changed